src/messaging/ExchangeMgr.cpp

```diff
@@ -198,8 +198,8 @@ void ExchangeManager::OnMessageReceived(const PacketHeader & packetHeader, const
     UnsolicitedMessageHandler * matchingUMH = nullptr;
     bool sendAckAndCloseExchange            = false;
 
-    ChipLogProgress(ExchangeManager, "Received message of type %d and protocolId %d", payloadHeader.GetMessageType(),
-                    payloadHeader.GetProtocolID());
+    ChipLogProgress(ExchangeManager, "Received message of type %d and protocolId %d on exchange %d", payloadHeader.GetMessageType(),
+                    payloadHeader.GetProtocolID(), payloadHeader.GetExchangeID());
```

Comment:
The exchange id is not really meaningful unless we're the initiator (i.e. the I flag is _not_ set here) or unless we also log the source node information for the message....  This is probably OK for now, but maybe a followup to have better logging here?

Reply:
Yeah this isn't very interesting by itself. I was using it with debug logs in send to figure out the lifetimes of exchanges while debugging more issues that lie beyond the one this PR is fixing.